benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py

```diff
@@ -0,0 +1,161 @@
+        "--verbose", action='store_true', help="Verbose runner output")
+    return parser.parse_args()
+
+
+def evaluate(env,
+             load_path="saved_files",
+             logs_path=None,
+             nb_episode=1,
+             nb_process=1,
+             max_steps=-1,
+             verbose=False,
+             save_gif=False,
+             **kwargs):
+    """
+    In order to submit a valid basline, it is mandatory to provide a "evaluate" function with the same signature as this one.
+
+    Parameters
+    ----------
+    env: :class:`grid2op.Environment.Environment`
+        The environment on which the baseline will be evaluated.
+
+    load_path: ``str``
+        The path where the model is stored. This is used by the agent when calling "agent.load)
+
+    logs_path: ``str``
+        The path where the agents results will be stored.
+
+    nb_episode: ``int``
+        Number of episodes to run for the assessment of the performance.
+        By default it's 1.
+
+    nb_process: ``int``
+        Number of process to be used for the assessment of the performance.
+        Should be an integer greater than 1. By defaults it's 1.
+
+    max_steps: ``int``
+        Maximum number of timestep each episode can last. It should be a positive integer or -1.
+        -1 means that the entire episode is run (until the chronics is out of data or until a game over).
+        By default it's -1.
+
+    verbose: ``bool``
+        verbosity of the output
+
+    save_gif: ``bool``
+        Whether or not to save a gif into each episode folder corresponding to the representation of the said episode.
+
+    kwargs:
+        Other key words arguments that you are free to use for either building the agent save it etc.
+
+    Returns
+    -------
+    ``None``
+    """
+    runner_params = env.get_params_for_runner()
+    runner_params["verbose"] = verbose
+
+    # Create the agent (this piece of code can change)
+    agent = RLAgent(env.action_space)
+
+    # Load weights from file (for example)
+    agent.load(load_path)
+
+    # Build runner
+    runner = Runner(**runner_params, agentClass=None, agentInstance=agent)
+
+    # you can do stuff with your model here
+
+    # start the runner
+    res = runner.run(
+        path_save=logs_path,
+        nb_episode=nb_episode,
+        nb_process=nb_process,
+        max_iter=max_steps,
+        pbar=False)
+
+    # Print summary
+    print("Evaluation summary:")
+    for _, chron_name, cum_reward, nb_time_step, max_ts in res:
+        msg_tmp = "\tFor chronics located at {}\n".format(chron_name)
+        msg_tmp += "\t\t - cumulative reward: {:.6f}\n".format(cum_reward)
+        msg_tmp += "\t\t - number of time steps completed: {:.0f} / {:.0f}".format(
+            nb_time_step, max_ts)
+        print(msg_tmp)
+
+    if save_gif:
+        save_log_gif(logs_path, res)
+
+
+if __name__ == "__main__":
+    """
+    This is a possible implementation of the eval script.
+    """
+    import grid2op
```

Comment:
Why don't you import them at the header ?

Reply:
copied from origin template. modified.